allenact/main.py

```diff
@@ -78,6 +82,14 @@ def get_argument_parser():
         help="experiment output folder",
     )
 
+    parser.add_argument(
+        "--save_dir_fmt",
+        required=False,
+        type=str,
+        default=SaveDirFormat.FLAT,
+        help="check SaveDirFormat",
+    )
+
```

Comment:
Now I see why you're using `.value` in places. I believe the following should work and keep things always as enums:
```python
    parser.add_argument(
        "--save_dir_fmt",
        required=False,
        type=lambda s: SaveDirFormat[s.upper()],
        default="flat",
        help="The file structure to use when saving results from allenact."
        " See documentation o f`SaveDirFormat` for more details."
        " Allowed values are ('flat' and 'nested'). Default: 'flat'.",
    )
```